core/src/main/java/com/cookpad/core/OkReport.kt

```diff
@@ -27,15 +27,22 @@ import com.cookpad.core.ui.OkReportActivity
 import github.nisrulz.screenshott.ScreenShott
 
 /**
+ * Call OkReport::trigger to show OkReport's screen.
+ */
+class OkReport(val trigger: () -> Unit) {
```

Comment:
I know I wrote this... 😅  

but now I look at it again, I think it's better to make the `constructor` **private**, so that the only way to make an instance of `OkReport` is through `initOkReport()`.